tests/tests/test_views.py

```diff
@@ -59,6 +59,30 @@ def test_pretty_names_from_filename(self):
         self.assertEqual(display_link.text.strip(), "test_molecule_no_context.html")
         self.assertEqual(render_link.text.strip(), pattern_path)
 
+    def test_pretty_names_from_filename_containing_dashes(self):
+        pattern_path = "patterns/molecules/test_molecule/test-molecule.html"
+        test_molecule_display_url = reverse(
+            "pattern_library:display_pattern",
+            kwargs={"pattern_template_name": pattern_path},
+        )
+        test_molecule_render_url = reverse(
+            "pattern_library:render_pattern",
+            kwargs={"pattern_template_name": pattern_path},
+        )
+
+        response = self.client.get(test_molecule_display_url)
```

Comment:
Would it make sense to also request the render url?

Reply:
done in c8ccc89